fuzz-test-suite/CMakeLists.txt

```diff
@@ -0,0 +1,21 @@
+# Determine the flags for fuzzing. Use OSS-Fuzz's configuration if available, otherwise fall back to defaults.
+if(DEFINED ENV{LIB_FUZZING_ENGINE})
+    set(FUZZING_ENGINE $ENV{LIB_FUZZING_ENGINE})
+    set(FUZZING_COMPILE_FLAGS "")
+    set(FUZZING_LINK_FLAGS "${FUZZING_ENGINE}")
+else()
+    set(FUZZING_COMPILE_FLAGS "-fsanitize=address,fuzzer")
```

Comment:
We already have a CI job for ASan, UBSan, and TSan. Any reason we don’t just add the fuzzer sanitizer there?

Reply:
I'll admit I missed that. For address sanitizer I think your right in that this can be removed here. However for the fuzzer sanitizer I think it should remain. The reason being that '-fsanitize=fuzzer' will provide its own main function which then calls LLVMFuzzerTestOneInput. The problem with enabling this project wide is it will try to link main functions to every binary which can lead to linking problems with multiple definitions of main in particular with unit tests. So this particular sanitizer should only be applied to fuzz harnesses. Docs here https://llvm.org/docs/LibFuzzer.html